Phonics/View Controllers/LettersViewController.swift

```diff
@@ -36,12 +43,21 @@ class LettersViewController: UIViewController, UICollectionViewDataSource, UICol
     //MARK: - Collection View Data Source
     
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return PHLetters.count
+        if difficulty == .easyDifficulty {
+            return PHLetters.count
+        }
+        
+        return phonics.count
     }
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "letter", for: indexPath) as! LetterCell
-        cell.decorateForLetter(PHLetters[indexPath.item], difficulty: difficulty)
+        if self.difficulty == .easyDifficulty {
+            cell.decorateForLetter(PHLetters[indexPath.item], difficulty: difficulty)
+        } else {
+            let phonic = phonics[indexPath.item]
```

Comment:
Just access it through the singleton instead. `PHContent.allPhonicsSorted[indexPath.item]`